tests/fixtures/joint.py

```diff
@@ -38,52 +38,89 @@
 QueryParams: TypeAlias = Mapping[str, int | str | list[str]]
 
 
+@dataclass
+class State:
+    """The current state of the database and the event topics"""
+
+    database_dirty: bool
+    events_dirty: bool
```

Comment:
It doesn't look like `events_dirty` is getting used.

Reply:
It's used only once in `reset_state`.

Reply:
Will be solved properly in #31.